Find a functionally equivalent code:android.animation.PropertyValuesHolder.setKeyframes:COMMENT
Method Modifier: public      
Comment:/**
 * Set the animated values for this object to this set of Keyframes.
 *
 * @param values One or more values that the animation will animate between.
 */

@@ -5,9 +5,9 @@
     for (int i = 0; i < numKeyframes; ++i) {
         keyframes[i] = (Keyframe) values[i];
     }
-    mKeyframeSet = new KeyframeSet(keyframes);
+    mKeyframes = new KeyframeSet(keyframes);
 }

